backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/image/RemoveImageCommandCallback.java

```diff
@@ -0,0 +1,82 @@
+package org.ovirt.engine.core.bll.storage.disk.image;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import javax.enterprise.inject.Typed;
+import javax.inject.Inject;
+
+import org.ovirt.engine.core.bll.tasks.CommandCoordinatorUtil;
+import org.ovirt.engine.core.bll.tasks.interfaces.CommandCallback;
+import org.ovirt.engine.core.common.action.RemoveImageParameters;
+import org.ovirt.engine.core.common.businessentities.AsyncTaskStatus;
+import org.ovirt.engine.core.compat.CommandStatus;
+import org.ovirt.engine.core.compat.Guid;
+import org.ovirt.engine.core.vdsbroker.vdsbroker.VDSErrorException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Typed(RemoveImageCommandCallback.class)
+public class RemoveImageCommandCallback implements CommandCallback {
+    private static final Logger log = LoggerFactory.getLogger(RemoveImageCommandCallback.class);
+
+    @Inject
+    private CommandCoordinatorUtil commandCoordinatorUtil;
+
+    @Override
+    public void doPolling(Guid cmdId, List<Guid> childCmdIds) {
+        RemoveImageCommand<RemoveImageParameters> command = getRemoveImageCommand(cmdId);
+        Set<Guid> taskIds = new HashSet<>(command.getReturnValue().getVdsmTaskIdList());
```

Comment:
We can create an issue afterwards to stick this somewhere, since I guess this is identical to DestroyImageCommandCallback

Reply:
Yep, I thought about this, but I figured that it's not the right time to touch other flows, even though it's a quite easy refactoring